.github/workflows/reusable-build-test-release.yml

```diff
@@ -176,9 +176,9 @@ jobs:
   meta:
     runs-on: ubuntu-latest
     outputs:
+      matrix_supportedSplunk: ${{ steps.matrix.outputs.supportedSplunk }}
```

Comment:
Should we also change that for `run-scripted-input-tests-full-matrix`? https://github.com/splunk/addonfactory-workflow-addon-release/blob/65fa96cd421005b08ae738d24e8968c41c4e1e9a/.github/workflows/reusable-build-test-release.yml#L1833

Reply:
Done, thanks